libs/model/test/thread/thread-lifecycle.spec.ts

```diff
@@ -19,14 +19,8 @@ import {
 import { AddressAttributes, R2_ADAPTER_KEY } from '@hicommonwealth/model';
 import * as schemas from '@hicommonwealth/schemas';
 import { TopicWeightedVoting } from '@hicommonwealth/schemas';
-import {
-  CANVAS_TOPIC,
-  getTestSigner,
-  sign,
-  toCanvasSignedDataApiArgs,
-} from '@hicommonwealth/shared';
 import { Chance } from 'chance';
-import { afterEach } from 'node:test';
+import { afterEach } from 'node:test'; // WARNING: sinon won't work if we import from vitest
```

Comment:
I think we should be using `vi` (Vitest mocking) instead of `sinon` and then it should work with Vitest native `afterEach`. We're mixing a lot of things here: vitest, node testing lib, and sinon so it's getting a little messy.